Delejez/Bosszúcska

```diff
@@ -0,0 +1,74 @@
+Italt mérgezni, csapba vérezni
+Lánccal folytani, bűzzel gyilkolni
+Anyád kurvája
+Csapba vérezni, italt mérgezni
+Lángok közt édes Klementin
+
+Dokit leszarni, csapba vérezni
+Áh ezaz *kacsint*, rajta túlleni
```

Comment:
A rajta túllenni az olyan mintha "getting over it" lenne, pedig amúgy eredetileg "getting on the brink". Javaslom, hogy "az élre jutni" vagy "közel kerülni" vagy "élre kerülni".